tutorial2/dojo-blog/src/App1.js

```diff
@@ -0,0 +1,12 @@
+import './App.css'
```

Comment:
filenames should always be in small letters

Reply:
done